### ISSUE TITLE
Error: Invalid value for "--filter_list": Filter list is invalid JSON: filtertype:none

### ISSUE THREAD

**Issue by samcro1967** [elastic/curator#1329]
Trying to use curator_cli to delete older indexes.  Cannot seem to get delete_indexes to work at all.

Show Indices works without issue and I get results. 

curator_cli --host localhost --port 9200 show_indices

Delete Indices does not seem to work no matter what I put for filer list.

curator_cli --host localhost --port 9200 delete_indices --filter_list '{"filtertype":"none"}'
Error: Invalid value for "--filter_list": Filter list is invalid JSON: filtertype:none

I am ruining curator_cli, version 5.6.0 on Windows 10 and the latest version of docker container sebp/elk.

**Comment by untergeek** [comment]
It's Windows. I haven't documented it yet, because you're only the second person I've encountered to have hit this snag. There's something about how the JSON bits are parsed on a Windows command line (`cmd` or PowerShell) that precludes it from being used.  The exact same line works just fine from a Linux or Mac machine, but won't on a Windows machine.

There may be something I'm missing, like there's some way to escape the characters to make them work in Windows. I haven't had a chance to try to chase that down, however.

Perhaps if there's someone who knows the Windows command-line, that knows how to escape characters they could shed some light on this subject for me.

**Comment by samcro1967** [comment]
Yeah, I tried adding back slash to escape the parenthesis.  I seems to display the desired text properly in the error message, but still errs out.

curator_cli --host localhost --port 9200 delete_indices --filter_list '{\"filtertype\":\"none\"}'
Usage: curator_cli delete_indices [OPTIONS]

Error: Invalid value for "--filter_list": Filter list is invalid JSON: '{"filtertype":"none"}'

**Comment by samcro1967** [comment]
Replacing the single quotes around the filtertype JSON and escaping the double quotes with a back slash inside the filtertype JSON seems to have worked.
C:\Program Files\elasticsearch-curator>curator_cli --host localhost show_indices --verbose --header --filter_list "{"filtertype":"age","source":"name","timestring":"%Y.%m.%d","unit":"days","unit_count":30,"direction":"older"}"
Index State Size Docs Pri Rep Creation Timestamp
telegraf-2018.12.13 open 103.8MB 1105152 5 1 2018-12-13T22:09:42Z
telegraf-2018.12.14 open 1.5GB 16806115 5 1 2018-12-14T00:00:00Z
telegraf-2018.12.15 open 1.5GB 17311397 5 1 2018-12-15T00:00:06Z
telegraf-2018.12.16 open 1.6GB 17499439 5 1 2018-12-16T00:00:08Z
telegraf-2018.12.17 open 1.6GB 17921179 5 1 2018-12-17T00:00:02Z
telegraf-2018.12.18 open 1.6GB 18098332 5 1 2018-12-18T00:00:06Z
telegraf-2018.12.19 open 1.6GB 18118504 5 1 2018-12-19T00:00:01Z
telegraf-2018.12.20 open 1.5GB 17397983 5 1 2018-12-20T00:00:04Z
telegraf-2018.12.21 open 1.3GB 16636635 5 1 2018-12-21T00:00:00Z
telegraf-2018.12.22 open 1.3GB 16824327 5 1 2018-12-22T00:00:05Z
telegraf-2018.12.23 open 1.3GB 17055585 5 1 2018-12-23T00:00:00Z
telegraf-2018.12.24 open 1.3GB 17043244 5 1 2018-12-24T00:00:03Z
telegraf-2018.12.25 open 1.1GB 14056419 5 1 2018-12-25T00:00:07Z
Switching to delete_indices seems to execute, but errored out.
C:\Program Files\elasticsearch-curator>curator_cli --host localhost delete_indices --filter_list "{"filtertype":"age","source":"name","timestring":"%Y.%m.%d","unit":"days","unit_count":30,"direction":"older"}"
2019-01-24 10:32:52,885 INFO Deleting selected indices: ['telegraf-2018.12.23', 'telegraf-2018.12.13', 'telegraf-2018.12.25', 'telegraf-2018.12.18', 'telegraf-2018.12.20', 'telegraf-2018.12.19', 'telegraf-2018.12.22', 'telegraf-2018.12.16', 'telegraf-2018.12.21', 'telegraf-2018.12.17', 'telegraf-2018.12.14', 'telegraf-2018.12.15', 'telegraf-2018.12.24']
2019-01-24 10:32:52,885 INFO ---deleting index telegraf-2018.12.23
2019-01-24 10:32:52,888 INFO ---deleting index telegraf-2018.12.13
2019-01-24 10:32:52,889 INFO ---deleting index telegraf-2018.12.25
2019-01-24 10:32:52,892 INFO ---deleting index telegraf-2018.12.18
2019-01-24 10:32:52,892 INFO ---deleting index telegraf-2018.12.20
2019-01-24 10:32:52,893 INFO ---deleting index telegraf-2018.12.19
2019-01-24 10:32:52,894 INFO ---deleting index telegraf-2018.12.22
2019-01-24 10:32:52,896 INFO ---deleting index telegraf-2018.12.16
2019-01-24 10:32:52,896 INFO ---deleting index telegraf-2018.12.21
2019-01-24 10:32:52,897 INFO ---deleting index telegraf-2018.12.17
2019-01-24 10:32:52,901 INFO ---deleting index telegraf-2018.12.14
2019-01-24 10:32:52,903 INFO ---deleting index telegraf-2018.12.15
2019-01-24 10:32:52,904 INFO ---deleting index telegraf-2018.12.24
2019-01-24 10:33:22,905 CRITICAL Failed to complete action: delete_indices. <class 'curator.exceptions.FailedExecution'>: Exception encountered. Rerun with loglevel DEBUG and/or check Elasticsearch logs for more information. Exception: ConnectionTimeout caused by - ReadTimeoutError(HTTPConnectionPool(host='localhost', port=9200): Read timed out. (read timeout=30))
delete_indices appearsd to have partially worked. I suspect this is a different issue and not related to MS DOS command line syntax errors. When I rerun show_indicies, the same ones are listed, but there size is all zero now with the exception of one of them.
C:\Program Files\elasticsearch-curator>curator_cli --host localhost show_indices --verbose --header --filter_list "{"filtertype":"age","source":"name","timestring":"%Y.%m.%d","unit":"days","unit_count":30,"direction":"older"}"
Index State Size Docs Pri Rep Creation Timestamp
telegraf-2018.12.13 open 0.0B 0 5 1 2018-12-13T22:09:42Z
telegraf-2018.12.14 open 0.0B 0 5 1 2018-12-14T00:00:00Z
telegraf-2018.12.15 open 0.0B 0 5 1 2018-12-15T00:00:06Z
telegraf-2018.12.16 open 0.0B 0 5 1 2018-12-16T00:00:08Z
telegraf-2018.12.17 open 1.6GB 17921179 5 1 2018-12-17T00:00:02Z
telegraf-2018.12.18 open 0.0B 0 5 1 2018-12-18T00:00:06Z
telegraf-2018.12.19 open 0.0B 0 5 1 2018-12-19T00:00:01Z
telegraf-2018.12.20 open 0.0B 0 5 1 2018-12-20T00:00:04Z
telegraf-2018.12.21 open 0.0B 0 5 1 2018-12-21T00:00:00Z
telegraf-2018.12.22 open 0.0B 0 5 1 2018-12-22T00:00:05Z
telegraf-2018.12.23 open 0.0B 0 5 1 2018-12-23T00:00:00Z
telegraf-2018.12.24 open 0.0B 0 5 1 2018-12-24T00:00:03Z
telegraf-2018.12.25 open 0.0B 0 5 1 2018-12-25T00:00:07Z
